galaxy_ng/app/management/commands/create-remote.py

```diff
@@ -78,7 +79,8 @@ def add_arguments(self, parser):
             "-r",
             "--requirements_file",
             type=self.load_requirements_file,
-            help="Either an abs path ending in .yaml|.yml to be loaded or the literal YAML string."
+            help="Either an abs path ending in .yaml|.yml "
```

Comment:
minor: probably can undo changes to management command files since this they are not related to the pr